scale_compute_vms.yml

```diff
@@ -82,49 +52,11 @@
       loop_control:
         loop_var: hyp
 
-    # create ssh key on the first hypervisor
-    - name: check if ssh key exists
-      stat:
-        path: "{{ ansible_ssh_key }}"
-      register: sshkey
-      delegate_to: "{{ groups.hypervisor|first }}"
-
-    - name:  Generate ssh key
-      shell:
-        ssh-keygen -q -N "" -f {{ ansible_ssh_key }}
-      when: sshkey.stat.exists == False
-      delegate_to: "{{ groups.hypervisor|first }}"
-
-    - name:  install sshpass on first hypervisor
-      package:
-        name: sshpass
-        state: installed
-      delegate_to: "{{ groups.hypervisor|first }}"
-
-    # copy first hypervisor key to all hypervisors (as vbmc ports are created on first hypervisor)
-    - name: remove key
-      shell: |
-        ssh-keygen -R {{ hyp }}
-      changed_when: false
-      with_items: "{{ groups.hypervisor }}"
-      loop_control:
-        loop_var: hyp
-      delegate_to: "{{ groups.hypervisor|first }}"
-      ignore_errors: true
-
-    - name: add ssh key
-      shell: |
-        echo '{{ ansible_ssh_pass }}' | sshpass ssh-copy-id -i {{ ansible_ssh_key }} -o 'StrictHostKeyChecking no' -f root@{{ hyp }}
-      changed_when: false
-      with_items: "{{ groups.hypervisor }}"
-      loop_control:
-        loop_var: hyp
-      delegate_to: "{{ groups.hypervisor|first }}"
     - name: set rhel 7 interfaces for hypervisors
```

Comment:
can you please update the name

Reply:
These changes are working one. I am afraid to touch anything. I will push another change fixing these typos. Thanks :)